tests/integration_tests/data_steward/validation/main_test.py

```diff
@@ -93,7 +93,7 @@ def _create_drug_class_table(bigquery_dataset_id):
                        destination_dataset_id=bigquery_dataset_id)
 
         # ensure concept ancestor table exists
-        if not bq_utils.table_exists(common.CONCEPT_ANCESTOR):
+        if not client.table_exists(common.CONCEPT_ANCESTOR):
```

Comment:
self.bq_client would not work because `_create_drug_class_table` is a static method.

Reply:
Yeah, it is non-static because there were no references to self. It could be made non-static by removing the decorator, adding self to the parameters and using `self.bq_client`.